README.md

```diff
@@ -2,4 +2,4 @@
 A filter cascade implementation in rust. This uses the 32 bit version of the MurmurHash3 algorithm and is specifically implemented for compatibility with a specific
 python filter cascade implementation (https://github.com/mozilla/crlite/blob/master/create_filter_cascade/FilterCascade.py).
```

Comment:
While you're at it, could you also replace this dead link with a permalink to the intended file (i.e. specific commit hash or tag instead of master)? If there is no specific tag, you can easily find the current revision for the commit by pressing <kbd>y</kbd> while viewing a file on Github.

This also applies to all other Github links in this PR.

Reply:
I rewrote the README and pointed to the tag for the lib.rs file where relevant.